var/spack/repos/builtin/packages/survey/package.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Survey(CMakePackage):
+    """Survey is a high-level performance tool product from Trenza, Inc.
+       The survey collector/analytics framework is a new generation,
+       high-level, light-weight multi-platform Linux tool set that
+       targets metric collection for high-level performance analysis
+       of applications running on both single node and on large-scale
+       platforms, including the Cray platforms.
+
+       The collector is designed to work on sequential, MPI, OpenMP,
+       and hybrid codes and directly leverages several interfaces
+       available for tools inside current MPI implementations including:
+       MPICH, MVAPICH, MPT, and OpenMPI. It also supports multiple
+       architectures and has been tested on machines based on Intel,
+       AMD, ARM, and IBM P8/9 processors and integrated GPUs.
+
+       Survey is a licensed product – source not openly available.
+       To access the survey source and build with spack please contact:
+       Trenza Inc. via: dmont@trenzasynergy.com or
+                        jeg@trenzasynergy.com
+    """
+
+    homepage = "http://www.trenzasynergy.com"
+    git      = "git@gitlab.com:trenza/survey.git"
+
+    maintainers = ['jgalarowicz']
+
+    version('master', branch='master')
+    version('1.0.1', branch='1.0.1')
+    version('1.0.0', branch='1.0.0')
+
+    variant('mpi', default=False,
+            description="Enable mpi, build MPI data collector")
+
+    # must have cmake at 3.12 or greater to find python3
+    depends_on('cmake@3.12:', type='build')
+
+    # for collectors
+    depends_on("libmonitor@2021.04.27+commrank", type=('build', 'link', 'run'))
+
+    depends_on("papi@5:", type=('build', 'link', 'run'))
+    depends_on("gotcha@master", type=('build', 'link', 'run'))
+    depends_on("llvm-openmp@9.0.0", type=('build', 'link', 'run'))
+
+    # MPI Installation
+    depends_on("mpi", when="+mpi")
+
+    depends_on("python@3:", type=('build', 'link', 'run'))
+    depends_on("py-setuptools", type='run')
+    depends_on("py-pip", type='build')
+    depends_on("py-python-dateutil", type='run')
+    depends_on("py-pandas", type='run')
+    depends_on("py-psutil", type='run')
+    depends_on("py-sqlalchemy", type='run')
+    depends_on("py-pbr", type=('build', 'run'))
+    depends_on("py-pyyaml", type='run')
```

Comment:
```suggestion
    depends_on("py-python-dateutil", type=('build', 'run'))
    depends_on("py-pandas", type=('build', 'run'))
    depends_on("py-psutil", type=('build', 'run'))
    depends_on("py-sqlalchemy", type=('build', 'run'))
    depends_on("py-pbr", type='build')
    depends_on("py-pyyaml", type=('build', 'run'))
```
pbr is usually only needed at build-time (it's used to read deps from a requirements.txt file instead of from setup.py/setup.cfg). The rest are likely needed at both build and run time. At build time, pip will search for these in the `PYTHONPATH` and try to download them from the internet if it can't find them. At run time, these are needed to run the Python code which imports these deps.

Reply:
@adamjstewart checked in changes to resolve this.